### PR TITLE
Docs: Initial Content for Sumo Logic

### PR DESCRIPTION
**Pull request by lorikrell**
*Issue #, if available:*
None at this time

*Description of changes:*
Created initial content for Sumo Logic including:

- Intro updates for Sumo Logic, new logo for badge
- Self Guided added new page for Sumo Logic
- Added folder and content for Observability and Monitoring Labs - 2 labs, with scenario intro
- Cleanup info about the lab environment
- README updated to help Sumo writers continue to contribute

I didn't change any of the config.toml settings or css.

I noticed the intro for Self Guided mentions IAM setup, but the repo and original repo with the base didn't have those files. Also need a Survey link.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Video of built site

https://user-images.githubusercontent.com/29388020/143139321-df109343-910f-4545-b3c3-4fb496251810.mp4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,135 @@
...
-## Create an additional flow/track/path
-To create an additional flow for your workshop to accommodate other AWS events you will need to replicate every file in /content so that Hugo will pick these up as part of a new language. We have pre-created flows in config.toml file for default and for EE (Event Engine). To create a flow for EE copy or link ```ln -s``` each file you want as part of that flow. Example 010_introduction/_index.md gets copied or linked to 010_introduction/_index.ee.md. ***Note*** there is no common files that show up in the EE flow. If you want a page to be available in a different flow there must be a file with ee.md extension, otherwise Hugo will not pick this file up.
+Each folder must have an _index.md file. The frontmatter (first lines) must include the following customized for your content:
 
-It is recommended to create the default flow first, then create the second flow after you have ensure the first flow works as expected.
+```
+---
+title: "Observability and Monitoring Lab"
```
Comment (dilipmrajan9) [inline README.md]: Can we please include "AWS" in front of Observability?

File: content/040_observability_monitoring_lab/40_scenario.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Lab 1: Find the Root Cause
+
+Here’s the scenario you are trying to troubleshoot and isolate the root cause. 
+
+TravelLogic is an app that facilitates travellers booking their flights online and provides the allied services including hotel booking, sight-seeing, and arranging visits to the well-known places at the destination. 
+
+You, a DevOps specialist, are supporting a TravelLogic app on AWS that uses following AWS services:
+
+- ELB (Application Load Balancer)
```
Comment (dilipmrajan9) [inline content/040_observability_monitoring_lab/40_scenario.md]: Can we expand ELB? And prefix with Amazon

File: content/040_observability_monitoring_lab/40_scenario.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Host/EC2
```
Comment (dilipmrajan9) [inline content/040_observability_monitoring_lab/40_scenario.md]: Expand EC2 ( Amazon Elastic Compute Cloud). Required for the first reference, further mentions of the service can be abbreviated.

File: content/040_observability_monitoring_lab/40_scenario.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- DynamoDB
```
Comment (dilipmrajan9) [inline content/040_observability_monitoring_lab/40_scenario.md]: Amazon DynamoDB

File: content/040_observability_monitoring_lab/40_scenario.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+#### The Problem to Solve:
+
+The TravelLogic app faces unusual latency in loading the app web page, also sometimes resulting in outage in the us-area-1. You need to find out the root cause of the latency and outages ASAP.
```
Comment (dilipmrajan9) [inline content/040_observability_monitoring_lab/40_scenario.md]: as soon as possible

File: content/040_observability_monitoring_lab/42_lab1.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+---
+title: "1. Find the Root Cause"
+chapter: true
+weight: 8
+---
+
+## Find the Root Cause
+
+The TravelLogic app that you support faces unusual latency in loading the app web page, also sometimes resulting in outage in the us-area-1. You need to find out the root cause of the latency and outages ASAP.
```
Comment (dilipmrajan9) [inline content/040_observability_monitoring_lab/42_lab1.md]: us-area-1 might be difficult for audience to understand. Can we say us-east-1? ASAP - as soon as possible or maybe something else like "as soon as the event occurs"

File: content/040_observability_monitoring_lab/42_lab1.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+    ![Entities tab](/images/lab-monitoring/lab1-rce-entities.png)
+
+1. Click the **Open In** button in the Entity section, and select **Entity Dashboard**, to open the relevant entity dashboard.
+
+    ![Entity Dashboard](/images/lab-monitoring/lab1-entity-dashboard.png)
+
+1. Open the relevant DynamoDB Events dashboard by clicking the **AWS DynamoDB - Events** option in the dropdown The data in the dashboard is fetched using the relevant metrics.
+
+    ![AWS DynamoDB Events](/images/lab-monitoring/lab1-dynamodb-events.png)
```
Comment (dilipmrajan9) [inline content/040_observability_monitoring_lab/42_lab1.md]: Only first reference of the service name needs to be "AWS" + Service name. So can remove AWS reference for future references. Pls update rest of the docs with the same.

**Review by dilipmrajan9 — CHANGES_REQUESTED** [review]
Hi Lori - Overall looks good but i have made some suggestions for AWS service naming protocol. Could you please make those changes and I can go ahead and merge?

**Comment by lorikrell** [comment]
Updated, thank you for the review feedback! I also did global searches for some of those items like ASAP, etc. If we have further updates, I'll send new PRs.

**Pull request merged**